helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -143,7 +158,13 @@ def create(self, vals_list):
                 vals["team_id"] = self._prepare_team_id(vals)
             if vals.get("user_id") and not vals.get("assigned_date"):
                 vals["assigned_date"] = fields.Datetime.now()
-        return super().create(vals_list)
+        res = super().create(vals_list)
+        for rec in res:
+            if rec.partner_id or rec.partner_email:
+                rec.send_partner_mail()
+                if rec.partner_id:
+                    rec.message_subscribe(partner_ids=rec.partner_id.ids)
```

Comment:
Please check how standard does this without overwriting the create, but using `thread` specific methods.

Reply:
The partner subscribe is now handled using `_message_auto_subscribe_followers` from `thread`, but the mail is still sent from the create method